docs/libcurl/curl_easy_setopt.3

```diff
@@ -357,6 +357,8 @@ Address of the sender. See \fICURLOPT_MAIL_FROM(3)\fP
 Address of the recipients. See \fICURLOPT_MAIL_RCPT(3)\fP
 .IP CURLOPT_MAIL_AUTH
 Authentication address. See \fICURLOPT_MAIL_AUTH(3)\fP
+.IP CURLOPT_MAIL_RCPT_IGNORE_INVALID
```

Comment:
Ok, naming is really hard and this is bikeshedding.

I got stuck on the word *invalid* here. The thing that curl ignores in this case is an error because SMTP signals that back for the `RCPT` command. It makes curl ignore those *errors*. Are those recipients necessarily always "invalid" ?

Plus, what if all such commands return error, surely curl won't consider that ok? Surely at least one needs to be fine?

How about naming this option  `CURLOPT_MAIL_RCPT_ALLLOWFAILS` ?

Reply:
Changed to `CURLOPT_MAIL_RCPT_ALLLOWFAILS`